### ISSUE TITLE
how long does it take to train the "textsum" model?

### ISSUE THREAD

**Issue by puppet101** [tensorflow/models#412]
[models/textsum]
I am trying to train the Gigaword dataset  with the default parameters which seems very slow.  But the loss is getting lower as the time goes by. Can anyone who have succeeded to train the model tell me how long it takes to make the model converge?  Thanks!

**Comment by andydavis1** [comment]
This is a question better suited for StackOverflow. Please ask it there and tag it with the `tensorflow` tag.
